corehq/apps/toggle_ui/views.py

```diff
@@ -30,10 +31,36 @@ def page_url(self):
         return reverse(self.urlname)
 
     @property
+    def show_usage(self):
```

Comment:
Nitpick: `show_usage` seems to be an action (like a function that would be called to do something). Consider renaming to `should_show_usage`.